library/std/src/sync/mutex.rs

```diff
@@ -107,8 +107,8 @@ use crate::sys::locks as sys;
 /// *guard += 1;
 /// ```
 ///
-/// It is sometimes necessary to manually drop the mutex guard to unlock it
-/// sooner than the end of the enclosing scope.
+/// It is sometimes necessary to manually drop the mutex guard or to create an inner scope
+/// to unlock it sooner than the end of the enclosing scope.
```

Comment:
The "to" fragments are stacked a little awkwardly here... maybe the sentence is better flipped?

```suggestion
/// To unlock a mutex guard sooner than the end of the enclosing scope,
/// either create an inner scope or drop the guard manually.
```